install_config/web_console_customization.adoc

```diff
@@ -1014,3 +1014,38 @@ xref:../install_config/install/advanced_install.adoc#advanced-install-configurin
 # See: https://docs.openshift.com/enterprise/latest/install_config/aggregate_logging.html
 #openshift_master_logging_public_url=https://kibana.example.com
 ----
+
+[[configuring-quota-notification-messages]]
+== Configuring Quota Notification Messages
+
+Whenever a user reaches or exceedes a quota a Quota Notification is put into the Notificaiton Drawer.
+A custom quota notification message, per quota type, can be added to the notificaiton.  For example:
+"Your project is over quota.  It is using 200% of 2 cores CPU (Limit). Upgrade to <a href='http://www.openshift.com'>
+OpenShift Pro</a> if you need additional resources.".  The "Upgrade to..." part of the notification is the custom
+message and may contain HTML/links to additional resources.
+
+. Create the following configuration scripts within a file (for example,
+*_quota-messages.js_*):
++
+[source, javascript]
+----
+// Set custom notificaiton messages per quota type/key
+window.OPENSHIFT_CONSTANTS.QUOTA_NOTIFICATION_MESSAGE = {
+    "pods": "Upgrade to <a href='http://www.openshift.com'>OpenShift Pro</a> if you need additional resources.",
+    "limits.memory": "Upgrade to <a href='http://www.openshift.com'>OpenShift Online Pro</a> if you need additional resources."
```

Comment:
nit: I'd switch to `'` for the outer strings so the HTML attributes have double quotes, which is more typical.

```
"pods": 'Upgrade to <a href="http://www.openshift.com">OpenShift Pro</a> if you need additional resources.',
"limits.memory": 'Upgrade to <a href="http://www.openshift.com">OpenShift Online Pro</a> if you need additional resources.'
```